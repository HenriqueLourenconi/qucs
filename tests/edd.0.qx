load legacy
load legacy/components
load legacy/qucsator
load plugins/misc
load plugins/verilog

options lang=leg_sch

<Components>
<EDD_ D1 1 10 140 -57 -113 1 2 "explicit" 0 "1" 0 "0" 1 "V1" 1>
<EDD_ D2 1 20 140 -57 -113 1 2 "explicit" 0 "1" 0 "0" 1 "V1" 1>
<EDD_ D3 1 30 140 -57 -113 1 2 "explicit" 0 "1" 0 "0" 1 "V1*2" 1>
</Components>

options lang=verilog_nl
list main
list EDD_*

options lang=qucsator
list main
list EDD_*
