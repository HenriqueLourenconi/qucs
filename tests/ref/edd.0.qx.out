Qacs : Qucs alternative command shell
used for testing only
This is free software, and you are welcome
to redistribute it under the terms of 
the GNU General Public License, version 3 or later.
See the file "COPYING" for details.
misc plugins: wip
legacy plugins: wip
module main();
EDD_1 #(.Type(explicit), .Branches(1), .I1(0), .Q1(V1)) D1(_net0, _net1);
EDD_1 #(.Type(explicit), .Branches(1), .I1(0), .Q1(V1)) D2(_net2, _net3);
EDD_2 #(.Type(explicit), .Branches(1), .I1(0), .Q1(V1*2)) D3(_net4, _net5);
endmodule // main

// paramset EDD_1
// paramset EDD_2
